themes/hugo-sustain/static/js/search.js

```diff
@@ -0,0 +1,59 @@
+var client = algoliasearch('QN236PQQTM', '0276fb4f826a5fd0554d70b20bd3741b');
```

Comment:
is this the API KEY by mistakes??

Reply:
Algolia provides 2 keys: admin and search. This is the search key that goes to the browser, it can only be used to perform searches. It's not really private, so there's no need to hide it on an env variable. Although it might be a good idea just to be consistent with the admin key.

The admin key, on the other hand, can be used to manage indexes (clear, add, etc.). This one is never sent to the browser.

Reply:
Algolia provides 2 keys: admin and search. This is the search key that goes to the browser, it can only be used to perform searches. It's not really private, so there's no need to hide it on an env variable. Although it might be a good idea just to be consistent with the admin key.

The admin key, on the other hand, can be used to manage indexes (clear, add, etc.). This one is never sent to the browser.

Reply:
Algolia provides 2 keys: admin and search. This is the search key that goes to the browser, it can only be used to perform searches. It's not really private, so there's no need to hide it on an env variable. Although it might be a good idea just to be consistent with the admin key.

The admin key, on the other hand, can be used to manage indexes (clear, add, etc.). This one is never sent to the browser.

Reply:
- Algolia provides 2 keys: admin and search. This is the search key that goes to the browser, it can only be used to perform searches. It's not really private, so there's no need to hide it on an env variable. Although it might be a good idea just to be consistent with the admin key.
- 
- The admin key, on the other hand, can be used to manage indexes (clear, add, etc.). This one is never sent to the browser.
- 
-

Reply:
- Algolia provides 2 keys: admin and search. This is the search key that goes to the browser, it can only be used to perform searches. It's not really private, so there's no need to hide it on an env variable. Although it might be a good idea just to be consistent with the admin key.
- 
- The admin key, on the other hand, can be used to manage indexes (clear, add, etc.). This one is never sent to the browser.
- 
-

Reply:
- Algolia provides 2 keys: admin and search. This is the search key that goes to the browser, it can only be used to perform searches. It's not really private, so there's no need to hide it on an env variable. Although it might be a good idea just to be consistent with the admin key.
- 
- The admin key, on the other hand, can be used to manage indexes (clear, add, etc.). This one is never sent to the browser.
- 
-

Reply:
Algolia provides 2 keys: admin and search. This is the search key that goes to the browser, it can only be used to perform searches. It's not really private, so there's no need to hide it on an env variable. Although it might be a good idea just to be consistent with the admin key.

The admin key, on the other hand, can be used to manage indexes (clear, add, etc.), so this is hidden and should never be shared.

Reply:
Fair enough - I thought it might be a mistake.  Just double checking.

Also GitHub had a bad day 2 days ago!

Reply:
Oh, wow, sorry for the comments spam.